brave/brave-ios
# Fix #5249: Onboarding Changes and Revamp

## Summary of Changes

This pull request fixes #5249

## Submitter Checklist:


## Test Plan:
- Fresh Install Brave
- Navigate through onboarding including set as default
- Enter cnn.com to check new callout shield after ntp callout

## Screenshots:


https://user-images.githubusercontent.com/6643505/165825898-3e221603-7b16-43b9-8ab9-f3b350be8465.mp4


![ntp](https://user-images.githubusercontent.com/6643505/165825915-199c5d26-27c4-498f-8891-80aed207c309.png)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
